libs/deal-ticket/src/components/order-dialog.tsx

```diff
@@ -72,7 +89,14 @@ export const OrderDialog = ({
         <p>{t(`Market: ${finalizedOrder.market.name}`)}</p>
       )}
       <p>{t(`Type: ${finalizedOrder.type}`)}</p>
-      <p>{t(`Amount: ${finalizedOrder.size}`)}</p>
+      <p>
+        {t(
```

Comment:
chore (non-blocking): i feel like these translate functions will break a lot of things when / if we switch over to a real translation library. this looks pretty much like what @lingui provides, except that the translation strings can't be interpolated like this, you usually need to provide the variables to it separately, so depending the language it can figure out where it needs to be inserted. 
probably not a big deal, but we could potentially minimise the future friction if we'd start using a real library instead. they usually just return plain english if there's no extra setup. maybe not super important right now tho.